blocks/editable/index.js

```diff
@@ -652,3 +633,9 @@ export default class Editable extends Component {
 Editable.contextTypes = {
 	onUndo: noop,
 };
+
+Editable.Value = function( { tagName: Tag = 'div', children, ...props } ) {
```

Comment:
It's a bit unfortunate we need to surface `tagName` as a prop, but it makes sense given that we need to assign `dangerouslySetInnerHTML` as a prop to the element, and can't otherwise return just the raw children. Hmmm!

My ideal:

```
<p><Editable.Value>{ content }</Editable.Value></p>
```

Reply:
> [...] can't otherwise return just the raw children. [...]

We can't? This worked: https://github.com/WordPress/gutenberg/compare/try/editable-state-tree#diff-c2353ca7ccb0e91040faf47baa55d644R175

Reply:
It works if it's an array structure but not when it's an HTML string

Reply:
Ah, right, you need an element to use the dangerous attribute.